lib/SILOptimizer/Mandatory/TFDifferentiation.cpp

```diff
@@ -4160,7 +4160,7 @@ static SILFunction *lookupOrSynthesizeGradient(ADContext &context,
       args.push_back(arg);
     // If it's not seedable, we need to create a default seed.
     if (!config.isSeedable()) {
-      auto seedTy = origTy->getSingleResult().getType();
+      auto seedTy = origTy->getResults()[config.getSourceIndex()].getType();
```

Comment:
Should `config.getSourceIndex` be `resultIndex`?

Reply:
resultIndex doesn’t exist here. It you meant “source” instead of “result”, I believe there’s a JIRA bug tracking this. Definitely need to change the wording.